smart_open/__init__.py

```diff
@@ -40,4 +36,8 @@
     'smart_open',
 ]
 
+
 __version__ = version.__version__
+
+logger = logging.getLogger(__name__)
```

Comment:
Why this revert? Isn't it safer to add the handler early on, to avoid issues like this in the future?

Reply:
I reverted it because it was no longer a necessary part of the solution.

If you think it's worth keeping for future-proofing the existing solution, then I can add it back in.

Reply:
Not critical, but it makes sense to add the handler as soon as possible. Who knows what the contributed modules will do in the future (a missed logging call on import…).